.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

```diff
@@ -0,0 +1,38 @@
+name: "Feature Request"
+description: Create a new ticket for a new feature request
+title: "[REQUEST]: "
+labels: ["enhancement", "triage"]
+
+body:
+  - type: textarea
+    id: summary
+    attributes:
+      label: "Summary"
+      description: Provide a brief explanation of the feature
+      placeholder: Describe in a few lines your feature request
+    validations:
+      required: true
+  - type: textarea
+    id: basic_example
+    attributes:
+      label: "Basic Example"
+      description: Provide an example or two (User Story).
```

Comment:
This is similar to my comment below, but a little easier. Maybe instead of asking for an example, ask them why they want to see that feature. There's a reason that they want to see this thing, and I think the original question is trying to get at that.